scripts/auto_create_input_config.py

```diff
@@ -0,0 +1,784 @@
+            new_assembly = {
+                "species": species["scientificName"],
+                "accession": accession,
+                "release_folder": species["releaseLink"] or None,
+                "taxonomy_id": species["taxonomyId"],
+                "variant_count": species["currentRs"],
+            }
+
+            eva_species[accession] = new_assembly
+
+    return eva_species
+
+
+def _tabix_list(path: str) -> Optional[List[str]]:
+    """Return list-of-seqnames via `tabix -l` or None if tabix fails."""
+    proc = subprocess.run(
+        ["tabix", "-l", path],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+    return proc.stdout.strip().splitlines() if proc.returncode == 0 else None
+
+
+def _header_contigs(path: str) -> List[str]:
+    """Read ##contig IDs by streaming the header (works with .vcf or .vcf.gz)."""
+    import gzip
+
+    opener = gzip.open if path.endswith(".gz") else open
+    contigs = []
+    with opener(path, "rt") as fh:
+        for line in fh:
+            if not line.startswith("##"):
+                break
+            if line.startswith("##contig=<ID="):
+                contigs.append(line.split("ID=")[1].split(",")[0].rstrip(">\n"))
+    return contigs
+
+
+def seq_region_matches(eva_file: str, ensembl_file: str) -> bool:
+    """
+    True if EVA and Ensembl VCFs share at least one seq-region.
+
+    - Tries tabix -l up to max_retries on the remote EVA VCF
+      (to manage API connection issues).
+    - If EVA seq-regions cannot be obtained after the retries, return False.
+    - Builds a .tbi for the local Ensembl VCF if it is missing.
+      If index build fails, falls back to parsing ##contig headers.
+    """
+
+    max_retries, retry_delay = 4, 60  # seconds
+
+    # ---------- EVA VCF (remote) ---------------------------------------
+    eva_seq = None
+    for attempt in range(1, max_retries + 1):
+        eva_seq = _tabix_list(eva_file)
+        if eva_seq:
+            break
+        sys.stderr.write(
+            f"[WARN] tabix -l attempt {attempt}/{max_retries} failed for {eva_file}\n"
+        )
+        if attempt < max_retries:
+            time.sleep(retry_delay)
+
+    if not eva_seq:
+        # Could not retrieve seq-regions for the remote VCF – give up here.
+        sys.stderr.write(
+            f"[WARN] Cannot obtain seq-regions for remote EVA file {eva_file}; "
+            "skipping comparison.\n"
+        )
+        return False
+
+    # Remove any EVA index file saved in CWD
+    for ext in (".tbi", ".csi"):
+        tmp = Path(os.path.basename(eva_file) + ext)
+        if tmp.exists():
+            tmp.unlink()
+
+    # ---------- Ensembl VCF (local) ------------------------------------
+    ens_seq = _tabix_list(ensembl_file)
+    if not ens_seq:
+        sys.stderr.write(f"[INFO] Building index for {ensembl_file}\n")
+        build = subprocess.run(
+            ["tabix", "-p", "vcf", ensembl_file],
```

Comment:
```suggestion
            ["tabix", "-p", "vcf", "-C", ensembl_file],
```

create CSI index, it is the safest option. Otherwise for some species like wheat which has large chromosome it will fail.